content/en/docs/setup/upgrade/helm/index.md

```diff
@@ -96,6 +96,32 @@ gateways is [actively in development](/docs/setup/upgrade/gateways/) and is cons
     $ helm upgrade istio-base manifests/charts/base -n istio-system --skip-crds
     {{< /text >}}
 
```

Comment:
I'm not sure we want to completely duplicate the content between here and the main canary upgrade page? Maybe we can describe the "concepts" on the canary page and then this page would just be how to use them with Helm. WDYT?

Reply:
That sounds MUCH better.

Reply:
Let's get a docs maintainer to agree to this before we do it. This will be in a later PR though.

Reply:
Install/upgrade is one of the most commonly complained about parts of Istio, so yes, I'm very keen to see this improve in a later PR :)